<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Bt_logar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGAR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_nome">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Seu username </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_senha">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Sua senha </string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;
font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TELA DE LOGIN &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>User </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Senha </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
